cmd/secrets-store-csi-driver/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	_ "net/http/pprof" // #nosec
 	"time"
 
+	"google.golang.org/grpc"
```

Comment:
nit: we should group the imports in the following order

```
import (
  stdlib

  internal packages

  external packages
)
```

We can do this cleanup in a follow up PR for all existing packages

Reply:
👍 agreed, I moved the grpc one but didnt move the others cause im not quite sure about `kubebuilder:scaffold:imports` as my goimports config moves some things around weirdly